cognite/neat/_graph/transformers/_value_type.py

```diff
@@ -64,3 +71,73 @@ def transform(self, graph: Graph) -> None:
                     graph.remove((s, property_uri, o))
                     new_property = URIRef(f"{property_uri}_{remove_namespace_from_uri(value_type_uri)}")
                     graph.add((s, new_property, o))
+
+
+class ConvertLiteral(BaseTransformer):
+    description: str = "ConvertLiteral is a transformer that converts a literal value."
```

Comment:
This convert essentially is data-typing literal values, probably converting strings to the best data type possible (e.g., bool, float, etc.). Therefore name of the transformer should highlight this.

Maybe: DatatypeLiterals 

```suggestion
    description: str = "ConvertLiteral is a transformer that improve data typing of a literal value."
```